enterprise/ssl-impl/src/test/java/io/crate/protocols/ssl/SslContextProviderTest.java

```diff
@@ -48,30 +48,6 @@ public static void beforeTests() throws IOException {
     }
 
     @Test
-    public void testClassLoadingFallback() {
```

Comment:
Yep that's a left-over of the initial implementation which had a fallback class depending on whether ssl was loaded or not.